src/Symfony/Component/Console/Tests/Helper/ProcessHelperTest.php

```diff
@@ -31,6 +31,28 @@ public function testVariousProcessRuns($expected, $cmd, $verbosity, $error)
         $this->assertEquals($expected, $this->getOutput($output));
     }
 
+    public function testMustRun()
+    {
+        $helper = new ProcessHelper();
+        $helper->setHelperSet(new HelperSet(array(new DebugFormatterHelper())));
+        $output = $this->getOutputStream(StreamOutput::VERBOSITY_VERBOSE);
+        $helper->mustRun($output, 'php -r "echo 42;"', null);
+
+        $this->assertEquals('', $this->getOutput($output));
```

Comment:
both `assertNull` and `assertEmpty` will not test for the type `string`, i.e. `null` value will pass both, while `null` won't pass `$this->assertEquals('', null)`

As the value here is either a `string` or `false` it should be `assertSame('')` (strict check) and not `assertEquals`